megamek/src/megamek/client/ui/swing/boardview/StepSprite.java

```diff
@@ -28,6 +33,16 @@
  * entering, exiting or turning.
  */
 class StepSprite extends Sprite {
+    
+    private final static GUIPreferences GUIP = GUIPreferences.getInstance(); 
```

Comment:
```suggestion
    private static final GUIPreferences GUIP = GUIPreferences.getInstance(); 
```

Reply:
Just standard order for this project.

Reply:
There is no standard order for this project. The Java Language Specification has a recommended order, but our style guide does not dictate any particular order.

Reply:
Sorry, standard order for the code within this project.

Reply:
??

Reply:
Technically the two formats are the same, however the code base tends to use the static final instead of final static so that's the better one to use for any new code, just to standardize the base.

Reply:
The order makes no difference.

Reply:
Code-wise it does not, however readability-wise it does. If I wrote `final static private int`, it would be a proper variable declaration, although it is not readable. Therefore, standardization helps the code be significantly more readable.

Reply:
It's perfectly readable that way. You're trying to impose your own preferences and inventing authorities to appeal to.

Reply:
That is abjectly wrong. It is not my preferences, it is standard Java practice and is part of our style guide: https://google.github.io/styleguide/javaguide.html#s4.8.7-modifiers

Reply:
That's not our style guide.

Reply:
So, my two cents on this (here specifically, and because it keeps coming up) is that a) changing the order of modifier declarations doesn't materially impact the readability of the code (unless it's a grep command), and b) having people spend their time making minor formatting changes is not really a productive use of said time. While I appreciate striving for code readability, the return on investment here is a lot less than the amount of time we've spent talking about it. To quote a wiser man than I, "the perfect is the enemy of the good".

Let's all please keep in mind that we're all doing this in our spare time as a hobby, and I'd rather the focus be on more "substantive" issues. Given that this is an intermediate PR, I'm going to merge it and assume our intrepid author will address the formatting issues in a subsequent one.